src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -265,6 +273,12 @@ contract QuestionAPI is Ownable, NFTLocked, FunctionLocked {
         emit QuestionDisqualified(questionId, _msgSender());
     }
 
+    function withdrawFromVault(uint256 questionId, STAGE stage) public {
```

Comment:
Why not just have the two above functions call this function and send in the appropriate stage - or I guess what's the use case for this one?